site/docs/hosting/cloudflare-workers-nodejs.md

```diff
@@ -18,37 +18,177 @@ If you're looking for the Deno version, please check out [this tutorial](./cloud
+
+::: tip
+You can change the name of your worker in `wrangler.toml` later.
+:::
+
+Next, you will be asked to select the type of your worker, here we choose `"Hello World" Worker`:
+
+```ansi{8}
+using create-cloudflare version 2.17.1
+
+╭ Create an application with Cloudflare Step 1 of 3
+│
+├ In which directory do you want to create your application?
+│ dir ./grammybot
+│
+╰ What type of application do you want to create?  // [!code focus]
+  ● "Hello World" Worker  // [!code focus]
+  ○ "Hello World" Worker (Python)  // [!code focus]
+  ○ "Hello World" Durable Object  // [!code focus]
+  ○ Website or web app  // [!code focus]
+  ○ Example router & proxy Worker  // [!code focus]
+  ○ Scheduled Worker (Cron Trigger)  // [!code focus]
+  ○ Queue consumer & producer Worker  // [!code focus]
+  ○ API starter (OpenAPI compliant)  // [!code focus]
+  ○ Worker built from a template hosted in a git repository  // [!code focus]
+```
+
+Next, you will be asked to choose whether you want to use TypeScript, if you want to use JavaScript, choose `No`.
+Here we choose `Yes`:
+
+```ansi{11}
+using create-cloudflare version 2.17.1
+
+╭ Create an application with Cloudflare Step 1 of 3
+│
+├ In which directory do you want to create your application?
+│ dir ./grammybot
+│
+├ What type of application do you want to create?
+│ type "Hello World" Worker
+│
+╰ Do you want to use TypeScript?  // [!code focus]
+  Yes / No  // [!code focus]
+```
+
+Your project will be set up in a few minutes.
+After that, you will be asked whether you want to use git for version control, choose `Yes` if you want the repo to be initialized automatically or `No` if you want to initialize it by yourself later.
+
+Here we choose `Yes`:
+
+```ansi{36}
+using create-cloudflare version 2.17.1
+
+╭ Create an application with Cloudflare Step 1 of 3
+│
+├ In which directory do you want to create your application?
+│ dir ./grammybot
+│
+├ What type of application do you want to create?
+│ type "Hello World" Worker
+│
+├ Do you want to use TypeScript?
+│ yes typescript
+│
+├ Copying template files
+│ files copied to project directory
+│
+├ Updating name in `package.json`
+│ updated `package.json`
+│
+├ Installing dependencies
+│ installed via `npm install`
+│
+╰ Application created
+
+╭ Configuring your application for Cloudflare Step 2 of 3
+│
+├ Installing @cloudflare/workers-types
+│ installed via npm
+│
+├ Adding latest types to `tsconfig.json`
+│ added @cloudflare/workers-types/2023-07-01
+│
+├ Retrieving current workerd compatibility date
+│ compatibility date 2024-04-05
+│
+╰ Do you want to use git for version control?  // [!code focus]
+  Yes / No  // [!code focus]
+```
 
-```asciiart:no-line-numbers
-.
-├── node_modules
-├── package.json
-├── package-lock.json
-├── src
-│   ├── index.js
-│   └── index.test.js
-└── wrangler.toml
+Finally, you will be asked whether you want to deploy your worker, choose `No`, we will deploy it when we have a working Telegram bot:
```

Comment:
```suggestion
Finally, you will be asked whether you want to deploy your worker, choose `No`, since we are going to deploy it when we have a working Telegram bot:
```